opentelemetry-sdk/src/opentelemetry/sdk/metrics/__init__.py

```diff
@@ -403,24 +409,61 @@ class MeterProvider(metrics_api.MeterProvider):
     Args:
         stateful: Indicates whether meters created are going to be stateful
         resource: Resource for this MeterProvider
+        shutdown_on_exit: Register an atexit hook to shut down when the
+            application exists
     """
 
     def __init__(
-        self, stateful=True, resource: Resource = Resource.create_empty(),
+        self,
+        stateful=True,
+        resource: Resource = Resource.create_empty(),
+        shutdown_on_exit: bool = True,
     ):
         self.stateful = stateful
         self.resource = resource
+        self._controllers = []
+        self._exporters = set()
+        self._atexit_handler = None
+        if shutdown_on_exit:
+            self._atexit_handler = atexit.register(self.shutdown)
 
     def get_meter(
         self,
         instrumenting_module_name: str,
         instrumenting_library_version: str = "",
     ) -> "metrics_api.Meter":
+        """See `opentelemetry.metrics.MeterProvider`.get_meter."""
         if not instrumenting_module_name:  # Reject empty strings too.
-            raise ValueError("get_meter called with missing module name.")
+            instrumenting_module_name = "ERROR:MISSING MODULE NAME"
+            logger.error("get_meter called with missing module name.")
         return Meter(
             self,
             InstrumentationInfo(
-                instrumenting_module_name, instrumenting_library_version
+                instrumenting_module_name, instrumenting_library_version,
             ),
         )
+
+    def start_pipeline(
+        self,
+        meter: metrics_api.Meter,
+        exporter: MetricsExporter = None,
+        interval: float = 15.0,
+    ) -> None:
+        """Method to begin the collect/export pipeline.
+        
+        Args:
+            meter: The meter to collect metrics from.
+            exporter: The exporter to export metrics to.
+            interval: The collect/export interval in seconds.
+        """
+        if not exporter:
+            exporter = ConsoleMetricsExporter()
+        self._exporters.add(exporter)
+        # TODO: Controller type configurable?
+        self._controllers.append(PushController(meter, exporter, interval))
+
+    def shutdown(self) -> None:
```

Comment:
I think this should unregister the atexit handler, in the same way that the pushcontroller does today.